nxt_editor/stage_view.py

```diff
@@ -683,6 +686,18 @@ def mousePressEvent(self, event):
                 # Any click on a graphics item that isn't selectable
                 super(StageView, self).mousePressEvent(event)
                 return
+            not_intractable = self.model.get_node_locked(item_path)
+            if not_intractable:
+                self._clicked_something_locked = True
+                # selection = self.scene().selectedItems()
+                # keep = []
+                # for i in selection:
+                #     _path = self.get_sel_path_for_graphic(i)
+                #     if _path and not self.model.get_node_locked(_path):
+                #         keep += [i]
+                # self.scene().selection
```

Comment:
Leftover testing or imporant?

Reply:
Was working on locking nodes, forgot to delete